### PR TITLE
📖 DEVELOPING - update browser support

### PR DESCRIPTION
**Pull request by aghassemi**

### CODE REVIEW

File: contributing/DEVELOPING.md [hunk 1]
```diff
@@ -70,11 +70,11 @@ For detailed information on testing, see [TESTING.md](TESTING.md).
 
 ## Supported browsers
 
-In general we support the 2 latest versions of major browsers like Chrome, Firefox, Edge, Safari, Opera, and UC Browser. We support desktop, phone, tablet and the web view version of these respective browsers.
+In general we support the 2 latest versions of major browsers like Chrome, Firefox, Edge, Safari, Opera, and UC Browser. We support desktop, phone, tablet and the web view version of these respective browsers. For iOS we support the latest 2 __major__ versions which covers about 2 years.
 
 Beyond that the core AMP library and builtin elements should aim for very wide browser support and we accept fixes for all browsers with market share greater than 1 percent.
 
-In particular, we try to maintain "it might not be perfect but isn't broken"-support for the Android 4.0 system browser and Chrome 28+ on phones.
+In particular, we try to maintain "it might not be perfect but isn't broken"-support for IE 11 and Chrome 41.
```
Comment (cramforce) [inline contributing/DEVELOPING.md]: So, uhm, what happens on the Android System browser? :)

Comment (aghassemi) [reply contributing/DEVELOPING.md]: I misread that and didn't notice "system browser" part, thought it was just saying we support Android 4.0. added back

File: contributing/DEVELOPING.md [hunk 2]
```diff
@@ -70,11 +70,11 @@ For detailed information on testing, see [TESTING.md](TESTING.md).
 
 ## Supported browsers
 
-In general we support the 2 latest versions of major browsers like Chrome, Firefox, Edge, Safari, Opera, and UC Browser. We support desktop, phone, tablet and the web view version of these respective browsers.
+In general we support the 2 latest versions of major browsers like Chrome, Firefox, Edge, Safari, Opera, and UC Browser. We support desktop, phone, tablet and the web view version of these respective browsers. For iOS we support the latest 2 __major__ versions which covers about 2 years.
 
 Beyond that the core AMP library and builtin elements should aim for very wide browser support and we accept fixes for all browsers with market share greater than 1 percent.
 
-In particular, we try to maintain "it might not be perfect but isn't broken"-support for the Android 4.0 system browser and Chrome 28+ on phones.
+In particular, we try to maintain "it might not be perfect but isn't broken"-support for IE 11, the Android 4.0 system browser and Chrome 41.
```
Comment (cramforce) [inline contributing/DEVELOPING.md]: Should we add older iOS versions here? They kind of fall under the same policy.

Comment (aghassemi) [reply contributing/DEVELOPING.md]: how about: + "iOS versions older than the latest 2"

**Comment by cramforce** [comment]
7 would be the absolute minimum. Also the minimum we see in practice, but I
haven't looked for ~6 months.

**Comment by erwinmombay** [comment]
we still see a lot of ios7

**Comment by aghassemi** [comment]
@cramforce @erwinmombay we have removed the iOS 8 viewport implementation (and Search does not surface AMP on iOS 8 anymore). Tested some canonical AMP pages in 8 and "not perfect but not too broken" describes it well. iOS 7 simulator is not even available for download anymore so I can't test.

Given iOS 7 is [less than 0.5% ](https://data.apteligent.com/ios/)at the moment and tools to test on it are not available anymore, let's do: _**"... for IE 11, iOS 8, the Android 4.0 system browser and Chrome 41 ..."**_

**Pull request merged**